APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
- | KYC Fill-in | KDDI | 2023/07/15 | [Link]([https://github.com/camaraproject/WorkingGroups/pull/268](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_KYC-Fillin_MTNandKDDI.md)) | DT, KDDI, MTN, TEF, GSMA Open Gateway, Orange, China Telecom | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
- | Site-to-cloud VPN | China Telecom, Huawei | 2023/07/23| [Link]([https://github.com/camaraproject/WorkingGroups/pull/273](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Site%20to%20cloud%20(S2C)%20VPN.md)) | China Telecom, Huawei, China Mobile | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
-  | Device Swap | MTN | 2023/08/13| [Link](https://github.com/camaraproject/WorkingGroups/pull/276) | MTN, GSMA Open Gateway, Huawei | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
- | IMEI Fraud | MTN | 2023/08/13| [Link](https://github.com/camaraproject/WorkingGroups/pull/277) | MTN, GSMA Open Gateway, Huawei | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
- | WebRTC | Telefónica | 2023/08/16| [Link]([https://github.com/camaraproject/WorkingGroups/pull/280](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_WebRTC_Telefonica.md)) | Telefónica, Mavenir, Orange, Vodafone, TMUS | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
-  | Network Insights | Verizon | 2023/08/31| [Link](https://github.com/camaraproject/WorkingGroups/pull/295) | Verizon, 5GFF, GSMA Open Gateway| YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
+
+
+
+## API backlog
+
+ This is a live doc that captures the status of all the APIs which have been formally registered in CAMARA project. 
+
+ | **API [family] name**  |  **API [family] owner** |  **API [family] proposal registration date**  | **Link to the filled-in application template**   |  **Supporters in API Backlog Working Group**  |  **TSC decision (date)** |  **Sub Project**  |  **API name**  |  **API documentation** |  **API version**  | **API definition**  |  **Implementation**  |  **API tested at** |  **API deployed at company and country**  |  
+ | --- |  --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |---| --- |
+ | Quality on Demand | DTAG | N/A | N/A | DTAG, Orange, Telefonica, Vodafone|  Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | qod | [Link](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/API_documentation) | v0.9.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions) | [PI_2 (Orange)](https://github.com/camaraproject/QualityOnDemand_PI2) | DTAG |  N/A |
```

Comment:
The documentation of the API is inline within the YAML, not within the linked directory. And: the documentation links should also point to the release branch to show the documentation which belongs to the released version.

```suggestion
 | Quality on Demand | DTAG | N/A | N/A | DTAG, Orange, Telefonica, Vodafone|  Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | qod | [Inline](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/release-0.9.0/code/API_definitions/qod-api.yaml) | v0.9.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions) | [PI_2 (Orange)](https://github.com/camaraproject/QualityOnDemand_PI2) | DTAG |  N/A |
```